src/gui/optionsdlg.ui

```diff
@@ -192,25 +197,211 @@
              </widget>
             </item>
             <item>
-             <widget class="QGroupBox" name="groupBox_4">
+             <widget class="QGroupBox" name="groupBoxAppearanceDesktop">
+              <property name="title">
+               <string>Desktop</string>
+              </property>
+              <layout class="QVBoxLayout" name="verticalLayout_27">
+               <item>
+                <widget class="QCheckBox" name="checkUseSystemTheme">
+                 <property name="text">
+                  <string>Use system icon theme</string>
+                 </property>
+                </widget>
+               </item>
+              </layout>
+             </widget>
+            </item>
+            <item>
+             <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Transfer List</string>
               </property>
-              <layout class="QVBoxLayout" name="verticalLayout_4">
+              <layout class="QVBoxLayout" name="verticalLayout_26">
                <item>
-                <widget class="QCheckBox" name="confirmDeletion">
+                <widget class="QCheckBox" name="checkAltRowColors">
                  <property name="text">
-                  <string>Confirm when deleting torrents</string>
+                  <string extracomment="In transfer list, one every two rows will have grey background.">Use alternating row colors</string>
```

Comment:
@evsh:  Grammar alert: 

>In transfer list, **one every two rows** will have grey background. 

should be 

>In transfer list, **every other row** will have **a** grey background.

Reply:
@evsh:  I am aware that you simply moved this code from elsewhere, but the English grammar in the original was also incorrect.